### ISSUE TITLE
Is there a way to "lock" the nodes in place? 

### ISSUE THREAD

**Issue by MSoup** [newcat/baklavajs#172]
I'm sorry if I didn't read enough. I'm looking for a way to freeze all the nodes in place. This will be particularly helpful when navigating a page with a lot of nodes. I haven't been able to figure out a solution myself yet.

**Comment by newcat** [comment]
Sorry for taking so long to respond. There is no real way to do this yet. I might be able to introduce a preventable event for moving nodes, will check that.

**Comment by LukasLoeffler** [comment]
@MSoup What do you mean by freezing all nodes?
Lock all nodes in place while the editor itself is movable?

**Comment by newcat** [comment]
Implemented in v1.10.0:

```js
viewPlugin.events.beforeNodeMove.addListener(..., () => false)
```
